### ISSUE TITLE
Non-decreasing Training Loss

### ISSUE THREAD

**Issue by GaoxiangLuo** [kakaoenterprise/Learning-Debiased-Disentangled#3]
Hi,

Thanks for open-sourced the code! I made a local copy of the code base as well as the CMNIST dataset. Simply by running the model of MLP and MLP_disentangled with provided Linux command lines as instructed in README. The training loss of either case is not decreasing and hence the test accuracy is similar with a naive classifier (~10%). Could you look into this issue?

Thank you!

**Comment by GaoxiangLuo** [comment]
It worked when I downgraded PyTorch 1.10.1 with CUDA 11.3 to PyTorch 1.7.1 with CUDA 11.0
